packages/devtools_app/lib/src/shared/console/widgets/display_provider.dart

```diff
@@ -75,10 +53,10 @@ class DisplayProvider extends StatelessWidget {
     }
 
     final hasName = variable.name?.isNotEmpty ?? false;
-    return DevToolsTooltip(
```

Comment:
is there a reason we removed the tooltip here?

Reply:
Because this was initially branched off of https://github.com/flutter/devtools/pull/6001 (which hasn't been submitted yet)